### PR TITLE
instance config.py may not be present if nothing needs to be overridden

### PR DESCRIPTION
**Pull request by ayan-usgs**

### CODE REVIEW

File: waterdata/__init__.py [hunk 1]
```diff
@@ -7,6 +7,9 @@
 
 # Loads configuration information from config.py and instance/config.py
 app.config.from_object('config')
-app.config.from_pyfile('config.py')
+try:
```
Comment (ayan-usgs) [inline waterdata/__init__.py]: Wondering if this is an appropriate thing to do... :thinking:

Comment (mbucknell) [reply waterdata/__init__.py]: I think it's fine. In the past, we always had tier specific settings, but perhaps we won't, particularly if we are using publically available endpoints

Comment (danielnaab) [reply waterdata/__init__.py]: I added a couple config settings for static assets - but you both will probably have feedback on how I set it up. I'm fine with this change though.

Comment (ayan-usgs) [reply waterdata/__init__.py]: @mbucknell -- Agreed. If we decide we need specific settings in the future, this doesn't preclude the use of a tier specific config.

Comment (jkreft-usgs) [reply waterdata/__init__.py]: and there will be tier-specific config, even if it is just boring stuff like a dev-specific google analytics code

**Pull request merged**